packages/controller-utils/src/constants.ts

```diff
@@ -84,3 +84,29 @@ export const OPENSEA_PROXY_URL =
   'https://proxy.metafi.codefi.network/opensea/v1/api/v1';
 export const OPENSEA_API_URL = 'https://api.opensea.io/api/v1';
 export const OPENSEA_TEST_API_URL = 'https://testnets-api.opensea.io/api/v1';
+
+/**
+ * Approval request types for various operations.
+ * These types are used by different controllers to create and manage
+ * approval requests consistently.
+ */
+export enum ApprovalType {
+  AddEthereumChain = 'wallet_addEthereumChain',
+  EthAccounts = 'eth_accounts',
```

Comment:
Didn't know about this one, which approval request uses this?

Reply:
It's not used, I just removed `EthAccounts` and `EthRequestAccounts`